include/dpp/guild.h

```diff
@@ -180,6 +180,8 @@ enum guild_member_flags : uint16_t {
 	gm_bypasses_verification = 0b0000000010000000,
 	/** Member has started onboarding */
 	gm_started_onboarding = 0b0000000100000000,
+	gm_roles_action    = 0b0000001000000000,
```

Comment:
Will this be sent to Discord? Will that cause issues?

Reply:
no, we serialise these into guild member data ourselves, our bitfields don't match discord's definitions